pyanaconda/modules/services/services.py

```diff
@@ -247,7 +247,8 @@ def install_with_tasks(self, sysroot):
                 disabled_services=self.disabled_services,
                 enabled_services=self.enabled_services
             ),
-
+            ConfigureSystemdDefaultTargetTask(sysroot=sysroot, default_target=self.default_target),
+            ConfigureDefaultDesktopTask(sysroot=sysroot, default_desktop=self.default_desktop),
```

Comment:
Ideally, this task shouldn't be created if `default_desktop` is not set. However, it is ok for now.

Reply:
Yeah, we have other tasks like this. We should handle this better later on.